.test-infra/jenkins/job_PostCommit_Python_ValidatesRunner_Flink.groovy

```diff
@@ -33,8 +33,8 @@ PostcommitJobBuilder.postCommitJob('beam_PostCommit_Python_VR_Flink',
   steps {
     gradle {
       rootBuildScriptDir(commonJobProperties.checkoutDir)
-      tasks(':beam-sdks-python:flinkCompatibilityMatrixBatch')
-      tasks(':beam-sdks-python:flinkCompatibilityMatrixStreaming')
+      tasks(':beam-sdks-python:flinkCompatibilityMatrixBatchProcess')
```

Comment:
Why not use a parameter instead? It would also be nice to move the sequence into the sdk gradle file so it is easier to run locally.

Reply:
+1 Appending arguments to the task name is not particularly readable. Parameters would make this more readable. The task `flinkCompatibilityMatrix` is already parameterized, so we just need to change the syntax here and remove the task generation code.

Reply:
Makes sense. And will also move the sequence to gradle task